### PR TITLE
Add toggle for Profilter Signals and Methods (+ update branch to 0.19.4)

### PR DESCRIPTION
**Pull request by pmessan**

### CODE REVIEW

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Build and Test
```
Comment (arthur-proglove) [inline .github/workflows/build_and_test.yml]: why is this file here? 
It's not present on our main branch https://github.com/workaroundgmbh/bleak/tree/proglove-v0.19.0

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "bleak-proglove"
-version = "0.19.0+profilter"
+version = "0.19.4+profilter"
```
Comment (arthur-proglove) [inline pyproject.toml]: I guess we can now have a `0.19.4.post1`

**Review by arthur-proglove — CHANGES_REQUESTED** [review]
Can you try to rebase on our `proglove-v0.19.0` instead? (we should call it `proglove-0.19.x` :-D)

File: .github/workflows/build_and_test.yml (hunk 1 above)
Comment (pmessan) [reply .github/workflows/build_and_test.yml]: I got confused during the rebase :D

File: pyproject.toml (hunk 2 above)
Comment (pmessan) [reply pyproject.toml]: Yes, yes we can :)

File: .github/workflows/build_and_test.yml (hunk 1 above)
Comment (pmessan) [reply .github/workflows/build_and_test.yml]: removed it now :)

File: CHANGELOG.rst [hunk 3]
```diff
@@ -41,6 +41,11 @@ Fixed
...
 * Fixed ``AttributeError`` crash when scanning on Windows builds < 19041. Fixes #1094.
 
+Fixed
+-----
+* BlueZ: Cancel the device discovery wait task if the device disconnects in
+  between to avoid a timeout
+
 Fixed
 -----
 * BlueZ: Cancel the device discovery wait task if the device disconnects in
```
Comment (arthur-proglove) [inline CHANGELOG.rst]: Something's wrong here, these added lines seem to exist already in the file?

Comment (pmessan) [reply CHANGELOG.rst]: How did that happen :D

Comment (pmessan) [reply CHANGELOG.rst]: Fixed it now. took the opportunity to cleanup the commits as well :)

File: bleak/backends/bluezdbus/manager.py [hunk 4]
```diff
@@ -194,25 +194,51 @@ async def async_init(self):
                 # Add signal listeners
 
                 bus.add_message_handler(self._parse_msg)
+                
+                uuids_list: List[str] = []
+                uuids = params.get('UUIDs')
+                if uuids is not None:
+                    uuids_list = uuids.value
```
Comment (arthur-proglove) [inline bleak/backends/bluezdbus/manager.py:201]: why `.value`? what is the type of `uuids`?

**Review by arthur-proglove — CHANGES_REQUESTED** [review]

File: bleak/backends/bluezdbus/manager.py (hunk 4 above)
Comment (pmessan) [reply bleak/backends/bluezdbus/manager.py:201]: it gets encoded as a dbus variant when the BleakScanner is instantiated

Comment (pmessan) [reply bleak/backends/bluezdbus/manager.py:201]: Added the type hint

Comment (arthur-proglove) [reply bleak/backends/bluezdbus/manager.py:201]: perfect :-)

**Review by arthur-proglove — APPROVED** [review]
